java/code/src/com/redhat/rhn/common/util/test/RpmVersionComparatorTest.java

```diff
@@ -168,6 +168,11 @@ public void testUbuntuBugzilla1150113() {
     public void testBugzilla1173201() {
         assertCompareSymm(-1, "1.27+1.3.9", "1.27.1ubuntu1+1.3.9");
         assertCompareSymm(-1, "1.27+1.3.9", "1.27.1+1.3.9");
+        assertCompareSymm(-1, "1.27+1.3.9", "1.27.1ubuntu2+1.3.11");
+        assertCompareSymm(1, "1.27+1.3.9", "1.3.11");
+        assertCompareSymm(1, "1.27", "1.3.11");
+        assertCompareSymm(1, "1.27+1.3.9", "1.3.9");
+        assertCompareSymm(-1, "1.27+1.3.9", "5.18.4.1");
```

Comment:
You adapted the check to test explicit of `~` not found, but no of this tests here has a case where a tilde exists.
Can you add such a test?

Reply:
Fixed